src/mumble/Settings.cpp

```diff
@@ -973,6 +974,7 @@ void Settings::load(QSettings *settings_ptr) {
 	LOAD(iTalkingUI_PrefixCharCount, "ui/talkingUI_PrefixCharCount");
 	LOAD(iTalkingUI_PostfixCharCount, "ui/talkingUI_PostfixCharCount");
 	LOAD(qsTalkingUI_AbbreviationReplacement, "ui/talkingUI_AbbreviationReplacement");
+	LOAD(qsTalkingUI_BackgroundColor, "ui/talkingUI_BackgroundColor")
```

Comment:
I'm not sure if these LOAD/SAVE functions will pass through CI as I haven't modified the ui file

Reply:
The CI is failing because of a typo elsewhere: